### PR TITLE
MWPFE-18 - added footer

### PR DESCRIPTION
**Pull request by lauramatiukievic**
Created a footer.

- added columns for info, address
- social media logos 
- copyright section
- added Links to needed information

### CODE REVIEW

File: src/app/layout.jsx [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+  description: "team website",
+};
 
 export default function RootLayout({ children }) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={inter.className}>
+        {children}
+
```
Comment (mbitc) [inline src/app/layout.jsx:17]: Looks good, but some don't using spaces :)

File: src/app/page.jsx [hunk 2]
```diff
@@ -1,24 +1,5 @@
 import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from "@/components/ui/accordion";
```
Comment (mbitc) [inline src/app/page.jsx:1]: This don't need, not using at this file.

File: src/app/page.jsx [hunk 3]
```diff
@@ -47,7 +47,8 @@ async function getPosts() {
 export default async function Home() {
   const { heroes } = await getProjects();
   const { posts } = await getPosts();
-  console.log(posts);
+
+  // console.log(posts);
```
Comment (dkisonas) [inline src/app/page.jsx:51]: remove

File: src/components/ui/footer.jsx [hunk 4]
```diff
@@ -1,112 +1,43 @@
 import React from "react";
 import Link from "next/link";
 import Image from "next/image";
-export default function Footer() {
+import FooterColumn from "./Footer-column";
+
+export default async function Footer({ data }) {
+  const { footerInfo, copyright, socialMediaIcon, footerColumn } = data;
+  console.log(socialMediaIcon);
```
Comment (dkisonas) [inline src/components/ui/footer.jsx:8]: remove

File: src/components/ui/footer.jsx [hunk 5]
```diff
@@ -1,112 +1,43 @@
...
+            <div className="flex-col sm:flex-col md:flex-col md:w-full lg:flex-row  xl:flex-row  flex items-start lg:gap-16 gap-24 self-stretch  ">
+              <div className=" flex  flex-col items-start ">
+                <div className="flex flex-row items-center ">
+                  <Image src={`${footerInfo.image.url}`} alt="logo" width="60" height="60" /> <h5>{footerInfo.name}</h5>
+                </div>
+                <p>{footerInfo.description}</p>
+              </div>
+              <div className=" grid grid-cols-2 grid-row-3  items-start gap-6  sm:flex sm:gap-6 md:flex md:gap-10 lg:flex lg:gap-6 xl:flex xl:gap-12 ">
+                {footerColumn.map((column) => (
+                  <FooterColumn data={column} key="footerColumn" title={column.title} links={column.links} />
```
Comment (dkisonas) [inline src/components/ui/footer.jsx:23]: you are passing column and column.title/links, choose one (I prefer column)

**Pull request closed without merging**